v1/rego/example_test.go

```diff
@@ -838,6 +854,7 @@ func ExampleRego_PrepareForPartial() {
 	if err != nil {
 		// Handle error.
 	}
+	makeStable(pqs.Queries)
```

Comment:
I'm mildly annoyed by this -- I hope this change is not a boomerang that'll hit us in the back of our heads when we least expect it. But it shouldn't be, ordering of rules returned by the RI isn't guaranteed, so I haven't tracked down where this is coming from. We have a bunch of map iterations in the ast/index.go code, so it could be any (or all) of those.